statchart/src/utils/get-color.ts

```diff
@@ -52,7 +52,7 @@ export function getStatChartColor(
 }
 
 function getColorFromMappings(value: StatChartValue, mappings: ValueMapping[]): string | null {
-  if (mappings?.length && value) {
+  if (mappings?.length && typeof value === 'number') {
```

Comment:
applyValueMapping accepts also strings, this could break intended functionality.

```suggestion
  if (mappings?.length && typeof value !== null) {
```